### ISSUE TITLE
Where is database for my settings, why am I having to reload my settings at GitHub

### ISSUE THREAD

**Issue by kevomac12** [uBlockOrigin/uBlock-issues#2836]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

My settings aren't saved on my laptop, why am I having to reload them... WT#

### A specific URL where the issue occurs.

https://www.monster.com/profile/error

### Steps to Reproduce

open Monster, ZipRecruiter, Forbes, SSA.gov, FB, Linkedin, 56 tabs in Edge... All must be loaded AGAIN in UBlock...

### Expected behavior

I expected to load a previously set URL and find the UBlock settings intact...

### Actual behavior

edge unloaded ublock origin, and I had to add the extension AGAIN, the settings in ublock GONE!

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Chromium: 117
filterset (summary):
 network: 97492
 cosmetic: 39646
 scriptlet: 18610
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  easylist: 71031-13, 1h.59m
  easyprivacy: 33173-63, 1h.59m
  plowe-0: 3719-1066, 1h.59m
  ublock-badware: 7879-142, 1h.59m
  ublock-filters: 34796-137, 1h.59m
  ublock-privacy: 556-6, 1h.59m
  ublock-quick-fixes: 165-64, 1h.59m
  ublock-unbreak: 2124-31, 1h.59m
  urlhaus-1: 3972-0, 1h.59m
filterset (user): [empty]
hostRuleset:
 added: [array of 185 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1648 ms (selfie)
 maxAssetCacheWait: 193 ms
```
</details>

**Comment by gwarser** [comment]
What settings? Did you lose your browsing session? Please describe in precise steps what exactly happened.

**Comment by kevomac12** [comment]
The browser was updated!

***************************************************
"When you pass through the waters, I will be with you: and when you pass through the rivers, they will not sweep over you. When you walk through the fire, you will not be burned: the flames will not set you ablaze."      
Isaiah 43:2
 

    On Monday, September 25, 2023 at 01:38:14 AM MDT, gwarser ***@***.***> wrote:  
 
 


What settings? Did you lose your browsing session? Please describe in precise steps what exactly happened.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
  

|  | Virus-free.www.avg.com |

**Comment by kevomac12** [comment]
How was the issue closed?If the browser updates, all settings are gone?

***************************************************
"When you pass through the waters, I will be with you: and when you pass through the rivers, they will not sweep over you. When you walk through the fire, you will not be burned: the flames will not set you ablaze."      
Isaiah 43:2
 

    On Monday, September 25, 2023 at 04:01:15 AM MDT, gwarser ***@***.***> wrote:  
 
 


Closed #2836 as completed.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
  

|  | Virus-free.www.avg.com |

**Comment by gothic-bum** [comment]
uBO settings are not retained when it is uninstalled/removed, which is what you said happened:  
  
> edge unloaded ublock origin, and I had to add the extension AGAIN, the settings in ublock GONE!
  
You need to investigate on your end why uBO was uninstalled/removed  (for example: incorrectly flagged by an antivirus program, malware installed on your PC, etc).
  
See:  
https://github.com/gorhill/uBlock/wiki/Software-known-to-have-uninstalled-uBlock-Origin
